01_fib/sarahdavy1983/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+// Write to this rather that Stdout directly to help with testing
+var out io.Writer = os.Stdout
+
+// fib returns a slice of integer fibonacci numbers
+func fib(n int) []int64 {
+
+	switch {
+	case n == 0:
+		return []int64{0, 0}
+	case n == 1:
+		return []int64{0, 1}
+	case n == -1:
+		return []int64{0, 1}
+	case n < -1:
+		x := fib(n + 1)
+		nextVal := x[len(x)-2] - x[len(x)-1]
+		return append(x, nextVal)
+	default:
+		x := fib(n - 1)
+		nextVal := x[len(x)-2] + x[len(x)-1]
+		return append(x, nextVal)
+	}
+
+}
+
+//printEachindex iterates over the slice and prints each index
+func printEachIndex(fibSlice []int64) {
```

Comment:
I'd name this `printIntSlice` or similar (nothing to do with fib imo) and have it print from index 0.
Pass it the appropriate slice with fib(n)[1:].
You could also rename your `fib` function to something like `fibSlice` because the required `fib` function was meant to have signature `fib(int n)`.
Then have a wrapper function `fib(int n)` that calls `printIntSlice(fibSlice(n)[1:])` or so

It's a bit a question of taste, so feel free to push back ;)